File: DesktopSample/DataBinding/PageAndRDL/JsonDataSource/C#/Web Service/Service.asmx.cs
using System.Web.Services;
using System.Web.Script.Services;

namespace ActiveReports.Samples.WebService
{
	[WebService(Namespace = "http://tempuri.org/")]
	[WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
	[System.ComponentModel.ToolboxItem(false)]
	[ScriptService]
	public class Service : System.Web.Services.WebService
	{
		[WebMethod]
		[ScriptMethod(ResponseFormat = ResponseFormat.Json)]
		public string GetJson()
		{
			return Properties.Resource.customers;
		}
	}
}

File: DesktopSample/DataBinding/PageAndRDL/JsonDataSource/VB.NET/Web Service/Service.asmx.vb
Imports System.Web.Services
Imports System.Web.Script.Services

<WebService(Namespace:="http://tempuri.org/")>
<WebServiceBinding(ConformsTo:=WsiProfiles.BasicProfile1_1)>
<CompilerServices.DesignerGenerated()>
<ComponentModel.ToolboxItem(False)>
<ScriptService()>
Public Class Service
    Inherits Services.WebService

    <WebMethod()>
    <ScriptMethod(ResponseFormat:=ResponseFormat.Json)>
    Public Function GetJson() As String
        Return My.Resources.Resource.customers
    End Function

End Class
